### ISSUE TITLE
Decompiler Error "$11330(24@,1600i) = &0(25@,1i)"

### ISSUE THREAD

**Issue by OrionSR** [sannybuilder/dev#29]
Sanny reports an incorrect jump instruction when decompiling this command:
`$11330(24@,1600i) = &0(25@,1i)`
I was just trying to verify that Sanny was compiling with 0084 and had to check with a hex editor instead.

The command is being used to copy parts of PS2 memory to expanded variable space so I can hunt for offsets, and it compiles and works as expected. I was just surprised that the command wouldn't decompile and wanted to drop a note on the issue.

**Comment by x87** [comment]
Sorry, can't reproduce. Tried both in SA and SA Mobile modes.

this:

```
{$CLEO}
$11330(24@,1600i) = &0(25@,1i)
```

gets decompiled to:

```
// This file was decompiled using SASCM.ini published by GTAG (http://gtag.gtagaming.com/opcode-database) on 6.14.2013
{$CLEO .cs}

//-------------MAIN---------------
0084: $11330(24@,64i) = &0(25@,1i) // (int) 
```

can you share a compiled file?

**Comment by OrionSR** [comment]
I don't think that's necessary. The problem I was having appears to be caused by compiling embedded scripts with an .es extension (so they don't accidentally get copied to the install folder or otherwise confused with cleo scripts). These scripts need to be renamed with .cs or the skip SCM header option needs to be enabled to decompile "external" scripts properly. 

I still think I was getting inconsistent results. Including extra info might be a factor in getting it to work right without skipping the header. Also, I might have been distracted by the false positives confounding the 010 script I'm using to convert local addresses to relative addresses in the embedded script. I've learned to avoid using 2@ as an array index. 

So, I've got a better handle on things now but will keep an eye out for other problems. If there's a problem with these arrays I'm bound to find it as I'm using them constantly in the embedded scripts.
